docs/contracts/abstract-contracts.rst

```diff
@@ -6,12 +6,15 @@
 Abstract Contracts
 ******************
 
-Contracts need to be marked as abstract when at least one of their functions is not implemented.
-Contracts may be marked as abstract even though all functions are implemented.
-
-This can be done by using the ``abstract`` keyword as shown in the following example. Note that this contract needs to be
-defined as abstract, because the function ``utterance()`` was defined, but no implementation was
-provided (no implementation body ``{ }`` was given).
+Contracts must be marked as abstract when at least one of their functions is not implemented or when
+their constructors (or declarations) do not provide arguments for all of their base contract constructors.
+A contract may be marked abstract even if it must not be marked abstract, such as when you do not intend
+for the contract to be created directly. Abstract contracts are similar to :ref:`Interfaces` but an
+interface is more limited in what it can declare.
```

Comment:
I'm marking this unresolved since it looks like it has not been changed in the code yet and you did not comment anything here.

To be clear, it's fine to disagree if you think this is not a good change, but please at least leave a comment saying why :)